sip-archetype/src/main/resources/archetype-resources/src/main/java/connectorgroups/__connectorGroup1__/connectors/GetAirQualityByCityInboundConnector.java

```diff
@@ -0,0 +1,60 @@
+package ${package}.connectorgroups.${connectorGroup1}.connectors;
+
+import ${package}.scenarios.definitions.GetAirQualityByCityScenario;
+import ${package}.scenarios.models.AirQualityResponse;
+import ${package}.scenarios.models.GeoCodingRequest;
+import de.ikor.sip.foundation.core.declarative.annonation.InboundConnector;
+import de.ikor.sip.foundation.core.declarative.connector.RestInboundConnectorBase;
+import de.ikor.sip.foundation.core.declarative.orchestration.Orchestrator;
+import de.ikor.sip.foundation.core.declarative.orchestration.connector.ConnectorOrchestrationInfo;
+import de.ikor.sip.foundation.core.declarative.orchestration.connector.ConnectorOrchestrator;
+import org.apache.camel.model.RouteDefinition;
+import org.apache.camel.model.rest.RestBindingMode;
+import org.apache.camel.model.rest.RestDefinition;
+import org.apache.camel.model.rest.RestParamType;
+
+/**
+ * Inbound connector which exposes GET /adapter/aircity/{city} endpoint.
```

Comment:
That comment should explain what functionality is provided by this connector. The path itself does not explain it very well.